### ISSUE TITLE
VS2017: Incorrect TS6082 when --outFile is specified

### ISSUE THREAD

**Issue by SaschaNaz** [Microsoft/TypeScript#15068]
**TypeScript Version:** VS2017 default 2.1.5.0

**Repro:**

1. Create a new Blank App (Universal Windows) project
2. Rename js/main.js to js/main.ts
3. Build+Deploy by clicking "Local Machine" and you will see js/main.ts below Script Documents
4. Go (Project) Properties->TypeScript Build
5. Write js/main.js on "Combine JavaScript output into file"
  ![image](https://cloud.githubusercontent.com/assets/3396686/24792225/13c7b620-1bb8-11e7-9e30-8e7962583ad7.png)
6. Build+Deploy
7. It will deploy successfully
8. Stop debugging and wait ~10 minutes

**Expected behavior:**

Nothing should happen, deploy should still succeed.

**Actual behavior:**

VS suddenly says

```
TS6082	Only 'amd' and 'system' modules are supported alongside --outFile
```

**Comment by minestarks** [comment]
@SaschaNaz this is known and we're working on it. In the meantime if you set your Module system explicitly to None you can work around it.

**Comment by minestarks** [comment]
Fix is in Visual Studio 15.2 , TS 2.3 and higher
